### PR TITLE
Added autoplay support

### PR DESCRIPTION
**Pull request by sbagdaulet**
Added autoplay support

### CODE REVIEW

**Comment by andrewvasilchuk** [comment]
@sbagdaulet, First off all thank you for your PR. Can you please explain the purpose behind it? The first thought that comes in mind is to play video when if first renders (maybe it was hidden behind `v-if`).

Maybe we should be more explicit and provide the `autoplay` prop,  because relying on `query` prop seems confusing to me (as you see [`autoplay`](https://github.com/andrewvasilchuk/vue-lazy-youtube-video/blob/master/src/VueLazyYoutubeVideo.vue#L72) is contained by default – it launches the `<iframe />`'s video when it's inserted into the DOM.

What do you think about it?

**Comment by sbagdaulet** [comment]
I think I understand your idea

I didn’t really understand the github, I apologize if I did something wrong.

File: src/VueLazyYoutubeVideo.vue [hunk 1]
```diff
@@ -129,6 +133,56 @@ export default Vue.extend({
         return ''
       }
     },
+    queryPart(): string {
```
Comment (andrewvasilchuk) [inline src/VueLazyYoutubeVideo.vue]: Why did you introduce query part? I think we can just leave `query` as computed property as is.

**Comment by andrewvasilchuk** [comment]
@sbagdaulet, There is a `query` [prop](https://github.com/andrewvasilchuk/vue-lazy-youtube-video#properties).

File: src/VueLazyYoutubeVideo.vue (hunk 1 above)
Comment (sbagdaulet) [reply src/VueLazyYoutubeVideo.vue]: autostarting a video for the background, for example, requires the video to be silent (before the user interacts with the site), if you pass the autoplay parameter to true but deny this in query then in this case the video will not be autostarted, so props autoplay is more priority, and this same parameter removes lazy loading, you need to work with it (as an option, start playback automatically if the video is in viewport)

Comment (sbagdaulet) [reply src/VueLazyYoutubeVideo.vue]: I’ll do a new pull request for lazy startup

**Comment by andrewvasilchuk** [comment]
@sbagdaulet, introduced in `v2.0.0-beta.0`. Closed via 1dbedc7b091ea55cf386f0012ff09adedce5524d.

**Comment by andrewvasilchuk** [comment]
@sbagdaulet, back-ported in  `1.4.0-beta.0`. Closed via 93858beef2f50701807252da0a29cbd321088c20.

**Pull request closed without merging**